exercicios/para-casa/biblioteca_pg2.py

```diff
@@ -0,0 +1,37 @@
+from livro import Livro
+
+class Biblioteca:
+    def __init__(self):
+        self.livros = []
+        
+    def adicionar_livro(self, livro: Livro):
+        if(not isinstance(livro, Livro)):
+            raise TypeError(f"Esperado livro obtido valor {livro} do tipo {type(livro)}")
+        self.livros.append(livro)
+
+    def exibir_livros(self, livro: Livro):
+        #if 
+        return self.livros
+    
+    def emprestar_livro(self, livro):
+        if livro in self.livros:
+            return True
```

Comment:
Esse método está indo no caminho certo porém aqui não basta retornar o True para o teste, o importante seria alterar a propriedade do livro de dentro da lista para emprestado = True

por exemplo: 
`self.livros[x].esta_emprestado = True`

x sendo a posição do livro que quer alterar de fato. Vc pode dar uma olhada na [função index()](https://www.freecodecamp.org/news/python-find-in-list-how-to-find-the-index-of-an-item-or-element-in-a-list/) para pegar a posição do item na lista